src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -1889,5 +1895,16 @@ private static string FormatPropertyDocumentation(TypeProperty property)
 
             return buffer.ToString();
         }
+
+        private static string? TryGetSymbolDocumentationMarkdown(Symbol symbol, SemanticModel model)
+        {   
+            if(symbol is DeclaredSymbol declaredSymbol && declaredSymbol.DeclaringSyntax is DecorableSyntax decorableSyntax)
+            {
+                var description = SemanticModelHelper.TryGetDescription(model, decorableSyntax);
+                var documentation = declaredSymbol is ParameterSymbol? $"Type: {declaredSymbol.Type.ToString() + MarkdownNewLine + description}" : description;
+                return documentation;
```

Comment:
[nit] `"Type: {declaredSymbol.Type.ToString() + MarkdownNewLine + description}"`
can be simplified to:
`"Type: {declaredSymbol.Type}{MarkdownNewLine}{description}"`